### DISCUSSION TITLE
incomplete release notes on renovate-bot/renovate-runner

### DISCUSSION THREAD

**Discussion by dhduvall** [renovatebot/renovate#23898] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

On the [renovate-runner project](https://gitlab.com/renovate-bot/renovate-runner) on GitLab, I'm often seeing in the merge requests release notes that are missing some of the versions. For instance, https://gitlab.com/renovate-bot/renovate-runner/-/merge_requests/1944 upgrades renovate from 36.46.0 to 36.48.0, but shows only the release notes for the transition from 36.47.0 to 36.48.0. The renovate repo does have three releases between 36.46.0 and 36.47.0; they're just not represented in the merge request.

I haven't been paying enough attention to figure out when this is happening, or when it started, though I don't remember seeing it before 36. And I haven't seen it in my own repos—at least, not yet.

This is unexpected to me, but is it a bug?

### What you tried so far.

Not my repo.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Example of a release which is missed: https://github.com/renovatebot/renovate/releases/tag/36.47.0

It _could_ be due to intentional truncation of release notes (exceeding the MR maximum length) but it doesn't seem like the MR is very long as-is

**Reply by rarkins** [reply]
Can you reproduce this in a repo on github.com?

**Reply by dhduvall** [reply]
Since I haven't seen it in a repo of my own, I wouldn't know where to start. If I do, though, I'll post something. Perhaps in the meantime the owners of the renovate-runner project can help track it down, assuming it's a high enough priority.

**Reply by rarkins** [reply]
Frankly it's not,
Unless it's discovered to be a widespread problem

**Reply by dhduvall** [reply]
Okay; it's low priority for me, too. I'll dig into it if I have the time and it annoys me sufficiently. I can also point out other instances if I run across them.
